.github/workflows/release.yml

```diff
@@ -1,13 +1,101 @@
 name: release
 
 on:
+  pull_request:
```

Comment:
I'm not sure we want to do this on pull request. If you want to test the windows stuff maybe break it out into its own action.

Reply:
That is how I had it originally, however, the release workflow needs to be able to copy the artifact from the msi build stage before uploading to the github release.

I could duplicate the stage into a new workflow but I think we should avoid having large duplicated workflows.

Reply:
I reverted that change, msi will only be built on release. 

The downside is not knowing if the msi build process is broken or not. If the msi step fails, the rest of the release pipeline will halt. 

Because workflows cannot depend on each other, the msi workflow must be coupled with goreleaser. We could de-couple it and have the msi workflow handle uploading the msi to the release, but this seems like a poor choice.

If we do want to build the MSI on PR, we could have it only run when MSI specific things are changed, such as the wix json, template files, etc.

Reply:
That's fair but I think that's an acceptable risk right now. If MSI fails and we can't release then that forces us to fix it before we release. It's not ideal but the risk of most code changes causing the MSI build to fail is likely low and running the entire MSI process on all PRs seems like over kill. 

If my above assumption turns out to be wrong I'm happy to come back to this.